MultiVersions/Forge/main/beautify/lang/zh_tw.json

```
@@ -0,0 +1,108 @@
+{
+	"itemGroup.beautify": "美化",
+
+	"tooltip.shift": "按住 SHIFT 查看更多資訊",
+	"tooltip.control": "按住 CTRL 查看可種植列表",
+
+	"block.beautify.rope": "繩索",
+	"rope.description1": "和鎖鏈一樣可放置、攀爬。",
+	"rope.description2": "可拉長懸吊花盆的繩索",
+
+	"block.beautify.hanging_pot": "懸吊花盆",
+	"hanging_pot.description1": "可懸吊在方塊、繩索下。也可以普通地放在地面上。",
+	"hanging_pot.description2": "可右鍵將植物種植到花盆內。",
+	"hanging_pot.description3": "可用骨粉催熟植物。用剪刀修剪後可獲得植物。",
+	"hanging_pot.list1": "可種植的植物：",
+	"hanging_pot.list2": "所有普通花朵和玫瑰叢、紫丁香、牡丹花、向日葵、藤蔓、垂泣藤、扭曲藤、發光地衣、螢光莓、甜莓、草、蕨",
+
+	"block.beautify.bookstack": "書堆",
+	"bookstack.description1": "按住 Shift 並對書堆右鍵可改變其排列方式。",
+	"bookstack.description2": "能增加附魔台的附魔等級",
+
+	"block.beautify.oak_picture_frame": "橡木畫框",
+	"block.beautify.birch_picture_frame": "樺木畫框",
+	"block.beautify.spruce_picture_frame": "衫木畫框",
+	"block.beautify.dark_oak_picture_frame": "黑橡木畫框",
+	"block.beautify.jungle_picture_frame": "叢林木畫框",
+	"block.beautify.acacia_picture_frame": "相思木畫框",
+	"block.beautify.warped_picture_frame": "扭曲蕈木畫框",
+	"block.beautify.crimson_picture_frame": "緋紅蕈木畫框",
+	"block.beautify.mangrove_picture_frame": "紅樹林木畫框",
+	"block.beautify.quartz_picture_frame": "石英畫框",
+	"frame.description1": "採隨機位置放置畫框",
+	"frame.description2": "按住 Shift 並對畫框右鍵可改變照片。",

```

純個人喜好：`Right Click` 翻作 `點擊右鍵`
```suggestion
	"frame.description2": "按住 Shift 並對畫框點擊右鍵，可改變照片。",

```